scripts/install

```diff
@@ -40,7 +39,7 @@ function strimzi {
         info "Installing Strimzi"
         helm upgrade strimzi-kafka-operator strimzi/strimzi-kafka-operator \
              --install \
-             --version 0.16.2 \
+             --version 0.17.0 \
```

Comment:
Is this a safe upgrade? Strimzi breaks contracts pretty regularly.

Reply:
/shrug
it's not critical

Reply:
do we have any data we care about if we're failing forward here though?

Reply:
You'll answer to @jmorris-pillar when you break prod, is all I'm saying.